components/desktop/pulseaudio/files/pulse_exec_attr

```diff
@@ -0,0 +1 @@
+Basic Solaris User:solaris:cmd:RO::/usr/bin/pulseaudio:privs=proc_priocntl,file_dac_read
```

Comment:
This means, it can read my /etc/shadow in any given time... Can we somehow alleviate this issue? Even setuid application looks more sane :)